src/inc/til/rle.h

```diff
@@ -589,7 +587,7 @@ namespace til // Terminal Implementation Library. Also: "Today I Learned"
 #ifdef UNIT_TESTING
         [[nodiscard]] std::wstring to_string() const
         {
-            std::wstringstream ss;
+            std::wstring str;

```

Comment:
if you're going to make this change in one place, we might as well make it everywhere in til! and maybe in a separate PR